ui/components/policy/page-counter.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function PageCounter({ count, page, pageSize }) {
```

Comment:
This component looks good and tested, but it doesn't look like its policy specific so it should be in `components/`. I think the benefit of this is that it is easy to find all the components in one spot, leading to increased chance of reuse and refactor.